packages/sanity/src/core/studio/components/navbar/search/components/filters/addFilter/AddFilterButton.tsx

```diff
@@ -12,7 +12,7 @@ import {AddFilterPopoverContent} from './AddFilterPopoverContent'
 export function AddFilterButton() {
   const [open, setOpen] = useState(false)
   const [buttonElement, setButtonElement] = useState<HTMLElement | null>(null)
```

Comment:
Could this also be turned into a ref now?

Reply:
No, `FilterPopoverWrapper` [doesn't support a ref](https://github.com/sanity-io/sanity/blob/bedbd5ad561aa7eab0324038dd2f49f708a86d19/packages/sanity/src/core/studio/components/navbar/search/components/filters/addFilter/AddFilterButton.tsx#L31).